osu.Game/Overlays/NotificationOverlayToastTray.cs

```diff
@@ -23,7 +24,9 @@ namespace osu.Game.Overlays
     /// </summary>
     public class NotificationOverlayToastTray : CompositeDrawable
     {
-        public bool IsDisplayingToasts => toastFlow.Count > 0;
+        public override bool IsPresent => toastContentBackground.Height > 0 || toastFlow.Count > 0;
+
+        public bool IsDisplayingToasts => allNotifications.Any();
```

Comment:
Is this correct anymore? It seems to have next to nothing to do with "toasts". One usage in this file was also changed to check `toastFlow.Count` instead of using this.

Reply:
Given its usage externally, it's still correct enough, but it can be changed to `toastFlow.Count` which may read better, so I'll do that.